src/main/java/org/cryptomator/cli/frontend/FuseMount.java

```diff
@@ -14,12 +16,16 @@
 	private static final Logger LOG = LoggerFactory.getLogger(FuseMount.class);
 
 	private Path vaultRoot;
+	private Path vaultPath;
```

Comment:
I think the `vaultPath` is not used anymore, could be deleted? (Also in the constructor below.)